tests/test.py

```diff
@@ -1045,6 +1046,44 @@ def execute_prepared_statement(self, query, args=None):
                 return data
 
 
+class Psycopg3Executor(QueryExecutor):
+    def _execute(self, query, args=None, prepare=None):
+        # We expect postgres placeholders, like $1, $2, $3...
+        # But psycopg expects only %s
+        # TODO: check that numbers are in the correct order
```

Comment:
they can be used not in ordered way. for example `insert into table1 values($1, $3, $1, $2, $3)`. It will depends on Bind packet and order of passed parameters

Reply:
Yeah, but psycopg don't have positional arguments. So, I think I will just create new array of arguments with duplications and the correct order